config/projects.yml

```diff
@@ -403,9 +405,9 @@ user-bundle:
         sonata_core: ['3']
         sonata_admin: ['3']
     3.x:
-      php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
+      php: ['5.6', '7.0', '7.1']
       versions:
-        symfony: ['2.3', '2.7', '2.8']
+        symfony: ['2.8']
```

Comment:
Please add symfony 3.3, so we have at least one symfony 3 release remaining

Reply:
Is there a proof PR for that? I'm not sure the user bundle is supposed to work with sf3 in version 2

Reply:
You're right. In that case 👍